patches/@playwright+test+1.31.2.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/node_modules/@playwright/test/lib/cli.js b/node_modules/@playwright/test/lib/cli.js
+index 7bf7a49..20dbb7a 100755
+--- a/node_modules/@playwright/test/lib/cli.js
++++ b/node_modules/@playwright/test/lib/cli.js
+@@ -161,6 +161,25 @@ async function runTests(args, opts) {
+   const status = process.env.PWTEST_WATCH ? await runner.watchAllTests() : await runner.runAllTests();
+   await (0, _profiler.stopProfiling)(undefined);
+   if (status === 'interrupted') process.exit(130);
++
++  const fs = require('fs');
```

Comment:
I think patching the library is the cleanest solution to track the actual computed snapshot file paths/names, and check for unused screenshots on exit. Otherwise, we'll have to pass an explicit filename when taking snapshots, call tracking code every time we take a snapshot, and have an extra script for detecting unused screenshots.

Manually tracking snapshot file paths would probably get cumbersome when we add additional interaction tests.

Reply:
The `toHaveScreenshot` matcher doesn't accept a file path, so manually constructing and tracking the snapshot file path isn't even really an option. I think I need the code in the library to capture the snapshot file paths.

Reply:
It's problematic and fragile to add a patch to a third-party library. I'm offering alternatives and solutions around it. I'm confident that we can find a way around this issue. Let's discuss some solutions around implementing a patch in our chat this afternoon.

Reply:
I understand that. I will look in this more to see if I can find a different solution that works without the problems I've mentioned.

Reply:
I have removed this patch.